parsers/uploader.py

```diff
@@ -35,15 +36,23 @@ def post(data, endpoint=UPLOAD_URL, token=UPLOAD_API_TOKEN):
                          headers={'API_TOKEN': token})
 
 
-def upload_datapoints(gen, upload_func=post):
-    """Save data from generator *gen* to database endpoint.
+def upload_datapoints(ust_data, upload_func=post):
+    """Save data from the ust data list by chunks to database endpoint.
     
        Returns:
           True on success (status code 200),
           False otherwise
     """
-    response = upload_func(data=to_json(gen))
-    if response.status_code == 200:
-        return True
-    else:
-        return False
+    ust_data_chunks = [ust_data[i:i + 1000] for i in range(0, len(ust_data), 1000)]
```

Comment:
I think 1000 should be a constant like chunck_size = 1000

Reply:
constants added